src/plays/sticky-notes/Readme.md

```diff
@@ -0,0 +1,27 @@
+# Sticky Notes
+
+Its a note taking app to take notes. so that you can use these notes to create content in the future.
+
+## Play Demographic
+
+- Language: js
+- Level: Beginner
+
+## Creator Information
+
+- User: coderamrin
+- Gihub Link: https://github.com/coderamrin
+- Blog: 
+- Video: 

```

Comment:
@Coderamrin Make sure to record a stream and add the stream link by editing the play.